pages/news/2020-05-16-intro-outro-sections.html

```diff
@@ -0,0 +1,91 @@
+There are no firm rules I use to decide which method to use; it's an artistic judgement based on the musical content of each track, what the vibe of the crowd is at that moment, and what I want to do with the mix.
+
+If I do not have the outro start point marked when I load a track, before I mix in the track, I skip ahead to the end of the track in my headphones to find the outro start point, then seek back to the intro start to get the track ready.
+If I don't have time to find the outro end point before mixing in the track, I can use the new deck cloning feature to drag and drop to a new deck, then seek ahead on the new deck and listen in my headphones to mark the outro start point.
+If I don't have time to do that either, I right click on the overview waveform where I think the outro starts and look how far that is from the end of the track to compare it to the length of the intro of the next track.
+<!-- screenshot -->
+
+### Intro & Outro Cues with AutoDJ ###
+
+In Mixxx 2.3, these new intro & outro section cues can be used to tell AutoDJ how long to crossfade and how to align the tracks in time.
+AutoDJ uses the intro & outro cues in the new "Full Intro + Outro" and "Fade At Outro Start" modes.
+In both modes, instead of crossfading over an arbitrary number of seconds, AutoDJ compares the duration of the outro of the old track and the intro of the new track.
+AutoDJ picks the time of the shorter section as the crossfade time.
+
+If the outro is shorter than the intro, AutoDJ will align the start of the outro with the start of the intro in both modes.
+<!-- screenshot -->
+
+The two modes work differently when the intro is longer than the outro.
+In the "Full Intro + Outro" mode, AutoDJ aligns the end of the intro and outro by starting the next track during the outro of the previous track.
+<!-- screenshot -->
+
+The "Fade At Outro Start" mode always aligns the start of the intro and outro.
+In this case, AutoDJ cuts off the end of the outro.
+This can be helpful if you want to prevent the energy of the mix from declining during a long outro.
+<!-- screenshot -->
+
+Both modes have been designed so that AutoDJ sounds good fading between any pair of tracks if the intro and outro sections have been marked reasonably.
+AutoDJ does not take into account the volume of each track, nor the frequency content, nor the rhythms, so it's not intended to be a replacement for a human DJ.
+However, it is good enough to give a human DJ a break without a major disruption to the mix.
+
+When there isn't a pandemic sweeping the globe, every week I DJ at a little gathering in my friend's yard while my friends spin fire.
+With the new AutoDJ modes, I can put two or three tracks in the AutoDJ queue with intro & outro sections marked and confidently step away from my gear for a few minutes so I can spin fire.
+It works well enough that no one even notices that the track changes but I'm not at my computer.
+
```

Comment:
Do we want to add a little "get involved" section here? Maybe tease future plans for track sections (GSoC) or AutoDJ and encourage users to help testing or something like that?

Reply:
Yes thanks for the reminder.